src/reconcile.ml

```diff
@@ -230,7 +291,20 @@ let reconcile ?styler (repls : Replacements.t) ~kind ~contents ~input_filename
               copy_input pos ~up_to:repl.start.pos_cnum ~line ~last_is_text
                 ~is_text
             in
-            let s = Replacement.text repl in
+            let s =
+              try Replacement.text ~use_compiler_pprint repl
+              with Astlib.Compiler_pprintast.Unavailable ->
+                let loc =
+                  {
+                    (Location.in_file input_filename) with
+                    loc_start = repl.start;
+                    loc_end = repl.stop;
+                  }
+                in
+                Location.raise_errorf ~loc
+                  "Ppxlib.Reconcile: Cannot print this AST fragment using the \
+                   compiler printers with OCaml < 4.14"
```

Comment:
Is this 4.14 going to have to be updated to 5.2 with the AST bump? If so, is there anyway for us to make this easier to remember?

Reply:
No, it's 4.14 as in the installed compiler version has to be at least 4.14 for this to work so this won't change for as long as we support OCaml < 4.14!